Ninjathulhu/Ninjathulhu.Game/Entity.cs

```diff
@@ -68,6 +68,30 @@ public class EntityProperties
         public Dictionary<Type, ComponentProperties> FromPrefab = new Dictionary<Type, ComponentProperties>();
 
         public Dictionary<Type, ComponentProperties> FromSpawn = new Dictionary<Type, ComponentProperties>();
+
+        private static object Get(Dictionary<Type, ComponentProperties> properties, Type componentType, string propertyName)
```

Comment:
I'm going to split out `EntityComponents` and `EntityProperties` into their own files, because it's getting too confusing to come across all these `Get` methods. PBP's pretty strict about "one class per file"; it works pretty well in their context, and might be too much file proliferation here, but I think it's worth a try. We can always re-combine them if it turns out to suck.